erpnext/templates/pages/integrations/authorizenet_checkout.html

```diff
@@ -3,49 +3,80 @@
 {% block title %} Payment {% endblock %}
 
 {% block script %}
-<script>{% include "templates/includes/integrations/authorizenet_checkout.js" %}</script>
 <script>
-	context= "{{ payment_context }}"
+	{
+		%
+		include "templates/includes/integrations/authorizenet_checkout.js" %
+	}
```

Comment:
Put this in one line, it's throwing a SyntaxError while checking out:

```suggestion
	{% include "templates/includes/integrations/authorizenet_checkout.js" %}
```